rest/taskrouter/statistics/workers/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,24 @@
+// Download the twilio-csharp library from
+// https://www.twilio.com/docs/libraries/csharp#installation
+using Newtonsoft.Json.Linq;
+using System;
+using Twilio;
+using Twilio.Rest.Taskrouter.V1.Workspace.Worker;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        const string workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+        TwilioClient.Init(accountSid, authToken);
+
+        WorkersStatistics stats = WorkerStatisticsResource.Fetch(workspaceSid);
```

Comment:
Usually is a good practice to rely on the C# type inference, please use `var` instead of a concrete type.